src/admin/mod.rs

```diff
@@ -19,12 +20,20 @@ use gperftools::profiler::PROFILER;
 
 #[cfg(feature = "gperftools")]
 use tokio::fs::File;
+use tokio::sync::{oneshot::Sender, Mutex};
 
 #[cfg(feature = "gperftools")]
 use tokio::io::AsyncReadExt;
 
 use crate::workload::WorkloadInformation;
-use tracing::info;
+use tracing::{error, info};
+
+lazy_static! {
+    /// Channel used to send shutdown signal - wrapped in an Option to allow
+    /// it to be taken by value (since oneshot channels consume themselves on
+    /// send) and an Arc<Mutex> to allow it to be safely shared between threads
+    static ref SHUTDOWN_TX: Arc<Mutex<Option<Sender<()>>>> = <_>::default();
```

Comment:
Why donot you instead pass a  mpsc channel to the quitquit handler

Reply:
Hi @hzxuzhonghu, I think the static `SHUTDOWN_TX` may be used by other threads in admin server where need to handle shutdown cases, then it can be re-used directly.

Reply:
Hi @hzxuzhonghu & @howardjohn, there are too long life time for admin server from `make_service_fn` and its `async move` to `service_fn` and its `async move`, so I suggest that a global static var can be used here for admin server's shutdown, so what do you think about it?

Reply:
Hi @hzxuzhonghu,  I figured out a way to remove such global static var, and both ztunnel and admin server can use the same shutdown signal. Re-subbmitted.

Reply:
LGTM